nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Updater/PackagesConfigUpdater.cs

```diff
@@ -255,18 +256,53 @@ private static Process[] GetLikelyNuGetSpawnedProcesses()
             if (hasPackage)
             {
                 // the dependency exists in the packages.config file, so it must be the second case
-                // the vast majority of projects found in the wild use this, and since we have nothing to look for, we'll just have to hope
-                partialPathMatch = "../packages";
+                // at this point there's no good way to determine what the packages path is, but there's a really good chance that it looks
+                // something like one of these:
+                //   ..\packages
+                //   ..\..\packages
+                //   etc.
+                // so first we'll see if we can find "packages\" somewhere in there
+                var genericHintPathNodes = projectBuildFile.Contents.Descendants().Where(IsHintPathNode).ToArray();
+                if (genericHintPathNodes.Length > 0)
+                {
+                    foreach (var hintPathNode in genericHintPathNodes)
+                    {
+                        // find the string "packages" followed by a directory separator...
+                        var hintPath = hintPathNode.GetContentValue();
+                        var match = Regex.Match(hintPath, @"^(.*?packages[/\\])");
```

Comment:
This was the real fix; if we didn't find the exact match, we can make a good guess by looking for some other `<HintPath>` that specifies `"packages"`.